### ISSUE TITLE
Node Certification Community Meeting

### ISSUE THREAD

**Issue by mrhinkle** [nodejs-certified-developer/certification#7]
Hello, 

I thought we'd try to get a meeting set-up to catch up with those people involved in the certification from earlier on and talk about plans going forward. I picked a time but if there needs to be another time we'll set up a Doodle.

I suggested March 15th at 12:00 Noon Eastern (UTC 5:00 p.m.). 

Node.js Foundation is inviting you to a scheduled Zoom meeting. 

Topic: Node.js Foundation's Certification Meeting
Time: Mar 15, 2018 12:00 PM Eastern Time (US and Canada)

Join from PC, Mac, Linux, iOS or Android: https://zoom.us/j/698718450

Or iPhone one-tap :
    US: +16465588656,,698718450#  or +16699006833,,698718450# 
Or Telephone:
    Dial(for higher quality, dial a number based on your current location): 
        US: +1 646 558 8656  or +1 669 900 6833  or +1 855 880 1246 (Toll Free) or +1 877 369 0926 (Toll Free)
    Meeting ID: 698 718 450
    International numbers available: https://zoom.us/zoomconference?m=YkruO6BhHUgsMEs86Bwub89WBxAGRxXl

**Comment by mrhinkle** [comment]
We met with the following folks to discuss the current state of the Node Certified Developer Exam on March 15th at 12:00 Noon Eastern: 

**Attending:**

Barry Woods (Nearform), David Clements (Nearform), Tierney Cyren (NodeSource), Jayco (Nodesource), Mark Hinkle (Executive Director, Node.js Foundation), Liz Kline (LF Training).

We are ready to test the HackerRank hosting environment.
Adding team for everyone to be pinged in the Github repo. 
Dave and Tierney will take a quick look at the existing questions. 
- Do we need to add anything to this question?
- Do we need to remove anything from this question?
- Can this question be answered according to any possible approach given the current version of Node?

Liz Kline - Creating a spreadsheet to track questions. 

Need to make sure we are looking at new features coming in Node 10.x and should they be covered
- e.g  HTTP2 

Reconvene next week to further discuss new invite coming.

**Comment by YerkoPalma** [comment]
Could the timeline be updated according to this info?

**Comment by mrhinkle** [comment]
@YerkoPalma we are still updating but we'll have an update soon.

**Comment by YerkoPalma** [comment]
Awesome, thanks for your work :)

**Comment by dresende** [comment]
GIthub didn't notify me about this ticket, only the #8 😢

**Comment by bnb** [comment]
@dresende yep, same - that's why I created the team do make sure that everyone can see these things effectively :grin

**Comment by dresende** [comment]
Thank you 🙏

**Comment by stefcameron** [comment]
This "team" concept is new to me. Where do I go to see this "team"?
